app/bundles/ReportBundle/Form/Type/ReportType.php

```diff
@@ -438,6 +440,16 @@ public function configureOptions(OptionsResolver $resolver)
             [
                 'data_class' => Report::class,
                 'table_list' => [],
+                'constraints' => [
```

Comment:
Why did you add constraint to the form itself? There is a custom validator which should catch this error.
What is the difference?
What happens when any other error rise? Bad email format, bad scheduler setting etc.?

Reply:
@Maxell92 Where is the custom validator? It must not be working for this to be reported as an issue. Maybe we were just missing adding the custom validator/constraint on the field?

Reply:
The issue is `error is on the non active tab`, right? So the validator have to work.
Here is the validation: https://github.com/mautic/mautic/blob/staging/app/bundles/ReportBundle/Scheduler/Validator/ScheduleIsValidValidator.php#L44

Reply:
The issue reported (internally) was that the form would save without an email address even if schedule is enabled. 

@davevurby Looks like that custom validator is applied in the Entity's loadValidatorMetadata at https://github.com/mautic/mautic/blob/1b283ba9fa37357de4e6e448817bf4ba5e9a9df4/app/bundles/ReportBundle/Entity/Report.php#L193.  Debug a bit around that. https://symfony.com/doc/2.8/validation/custom_constraint.html may be helpful on how custom validators work.

Reply:
Strange, pretty sure that validator has worked. But the description of this PR says that validation works, only tab with error is not active and report is not saved.
@davevurby Try to checkout code right after Report PR was merged if validator really fails to catch errors. There are more validations in validator, not only this one.